---
timestamp: '2025-01-08T00:11:22.026148'
level: INFO
issuer: instance_solver
event_type: metadata
data:
    log_file: exp_logs_anu/bfs relax-all lp1-new/mprime__pprob26-err-rate-0-5.yaml
    instance_id: mprime__pprob26-err-rate-0-5
    search_algorithm: bfs
    plan_length: 6
    domain_class: mprime
    instance_name: pprob26-err-rate-0-5
    ground_plan:
    - (overcome boils entertainment haroset earth neptune)
    - (feast entertainment haroset marzipan oregon bavaria)
    - (drink rice marzipan oregon manitoba quebec moravia kentucky)
    - (feast entertainment marzipan turkey moravia kentucky)
    - (feast entertainment turkey papaya manitoba oregon)
    - (succumb boils entertainment papaya earth neptune)
    lifted_plan:
    -   - overcome
        - ?boils
        - ?entertainment
        - ?haroset
        - ?earth
        - ?neptune
    -   - feast
        - ?entertainment
        - ?haroset
        - ?marzipan
        - ?oregon
        - ?bavaria
    -   - drink
        - ?rice
        - ?marzipan
        - ?oregon
        - ?manitoba
        - ?quebec
        - ?moravia
        - ?kentucky
    -   - feast
        - ?entertainment
        - ?marzipan
        - ?turkey
        - ?moravia
        - ?kentucky
    -   - feast
        - ?entertainment
        - ?turkey
        - ?papaya
        - ?manitoba
        - ?oregon
    -   - succumb
        - ?boils
        - ?entertainment
        - ?papaya
        - ?earth
        - ?neptune


---
timestamp: '2025-01-08T00:11:22.029738'
level: INFO
issuer: instance_solver
event_type: ground_repair
data:
    repair_length: 0
    repair: ''


---
timestamp: '2025-01-08T00:20:37.718066'
level: ERROR
issuer: batch_solver
event_type: error
data:
    error_type: MemoryError
    error_message: 'Command ''[''/home/projects/u7899572/conda-envs/main/bin/python3'',
        ''instance_solver.py'', ''bfs'', ''input/benchmarks-G1'', ''exp_logs_anu/bfs
        relax-all lp1-new/mprime__pprob26-err-rate-0-5.yaml'', ''1000000'', ''mprime__pprob26-err-rate-0-5'',
        ''1.0'', ''zeroary'']'' died with <Signals.SIGABRT: 6>.'
    stack_trace: "Traceback (most recent call last):\n  File \"batch_solver.py\",\
        \ line 119, in worker\n    result = subprocess.run(cmd, check=True, timeout=params['timeout_seconds'],\n\
        \  File \"/home/projects/u7899572/conda-envs/main/lib/python3.8/subprocess.py\"\
        , line 516, in run\n    raise CalledProcessError(retcode, process.args,\n\
        subprocess.CalledProcessError: Command '['/home/projects/u7899572/conda-envs/main/bin/python3',\
        \ 'instance_solver.py', 'bfs', 'input/benchmarks-G1', 'exp_logs_anu/bfs relax-all\
        \ lp1-new/mprime__pprob26-err-rate-0-5.yaml', '1000000', 'mprime__pprob26-err-rate-0-5',\
        \ '1.0', 'zeroary']' died with <Signals.SIGABRT: 6>.\n"
    instance_id: mprime__pprob26-err-rate-0-5
    process_info:
        cmd:
        - /home/projects/u7899572/conda-envs/main/bin/python3
        - instance_solver.py
        - bfs
        - input/benchmarks-G1
        - exp_logs_anu/bfs relax-all lp1-new/mprime__pprob26-err-rate-0-5.yaml
        - '1000000'
        - mprime__pprob26-err-rate-0-5
        - '1.0'
        - zeroary
        duration: '557.09'
        python_executable: /home/projects/u7899572/conda-envs/main/bin/python3
    stdout: ''
    stderr: 'terminate called after throwing an instance of ''Minisat22::OutOfMemoryException''

        '


